cluster-scope/overlays/moc/zero/groups/apicurio.enc.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: user.openshift.io/v1
```

Comment:
This group resource is encrypted correctly, however there are few changes needed:

1. Can you please relocate it to `cluster-scope/overlays/moc/common/groups/apicurio.enc.yaml`? The change is `s/zero/common/` in the path. It will essentially make the user group is available on all MOC clusters allowing you simpler migration and ArgoCD management. 
2. Additionally please include the resource in [`cluster-scope/overlays/moc/common/secret-generator.yaml`](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/common/secret-generator.yaml).
3. Please change the usernames mentioned in `users` list of the user group from GitHub nicks to MOC SSO usernames - it's the email address you use whey you log into the web console (https://console-openshift-console.apps.zero.massopen.cloud/ ) using `moc-sso` oauth. We currently don't have GitHub auth enabled so the usernames you've listed here won't ever get assigned.

Reply:
done